mllib/src/main/scala/org/apache/spark/ml/evaluation/RegressionEvaluator.scala

```diff
@@ -49,8 +49,8 @@ final class RegressionEvaluator @Since("1.4.0") (@Since("1.4.0") override val ui
    */
   @Since("1.4.0")
   val metricName: Param[String] = {
-    val allowedParams = ParamValidators.inArray(Array("mse", "rmse", "r2", "mae"))
-    new Param(this, "metricName", "metric name in evaluation (mse|rmse|r2|mae)", allowedParams)
+    val allowedParams = ParamValidators.inArray(Array("mse", "rmse", "r2", "r2adj", "mae"))
+    new Param(this, "metricName", "metric name available (mse|rmse|r2|r2adj|mae)", allowedParams)
```

Comment:
Not sure you want to change "in evaluation" here

Reply:
How about "valid metrics" instead?

Reply:
This is simply because the length of line is > 100 if I keep "in evaluation". I am ok to keep it.